public/js/keyboard.js

```diff
@@ -12,11 +12,13 @@ var moveKeys = {
         '187': 'equals',
         '189': 'minus'
     },
-    shift = false,
+    KeyBoard = {};
+
+let shift = false,
     buffer = [],
+    //the next ones are not used
```

Comment:
Are you saying that `xDown` and `yDown` are never used?
Don't add this commentary, just remove the variables.
If you want you can make another PR for that, since this is not related, but don't add this comment.

Reply:
Yes
Ok, I can make another PR about this and discuss about those variables later, for now I'll remove the commentary